api-spec/api-spec.md

```diff
@@ -0,0 +1,47 @@
+
+### WFS3 Endpoints
+
+The core WFS 3 endpoints are shown below, with details provided in an [OpenAPI specification document](definitions/WFS3.yaml).
+
+| Endpoint      | Returns          | Description        |
+| ------------ | ------------- | ---------------------- |
+| / | JSON        | Landing page, links to API capabilities |
+| /conformance | JSON | Info about standards the API conforms to       |
+| /collections | Collections | List of Collections contained in the catalog |
+| /collections/{collection_id} | Collection | Returns single Collection JSON |
+| /collections/{collection_id}/items | Items | GeoJSON FeatureCollection of Items in Collection |
+| /collections/{collection_id}/items/{item_id} | Item | Returns single Item GeoJSON |
+
+The `/collections/{collection_id}/items` endpoint accepts parameters for filtering the results (also called filters). Searching using POST will accept a JSON object where the top level keys specify which type of filter to apply (e.g., bbox=[...]). Those same key names can be used as query string parameters for the GET request. 
+
+Items in the collection should match all filters to be returned when querying. This implies a logical AND operation. If an OR operation is needed, it should be specified through an extension filter.
+
+### Filter Parameters
+
+| Parameter      | Type          | Description        |
+| ------------ | ------------- | ---------------------- |
+| bbox | [number] (x4)        | Requested bounding box [west, south, east, north] |
+| time | datetime string | Single date, date+time, or a range ('/' seperator) |
+| intersects | GeoJSON Feature | Searches items by performing intersection between their geometry and provided GeoJSON Feature |
+| page | number | The page number of results. Defaults to 1) |
+| limit | number | The maximum number of results to return (page size). Defaults to 10 |
+
+## STAC Endpoints
+
+STAC provides some additional endpoints for the root Catalog itself, as well as the capability to search the Catalog. Note that a STAC API does not need to implement WFS 3, in which case it would only support the endpoints given below. See the [OpenAPI specification document](definitions/STAC-standalone.yaml).
+
+```
+| Endpoint      | Returns          | Description        |
+| ------------ | ------------- | ---------------------- |
+| /stac | Catalog        | Root catalog |
+| /stac/search | Items | GeoJSON FeatureCollection of Items found |
+```
+
+The `stac/search` endpoint is similar to the `items` endpoint in WFS3 in that it accepts parameters for filtering, however it performs the filtering across all collections. The parameters accepted are the same as the Filter Parameters above, however the *[extensions](extensions/README.md)* also provide advanced querying parameters.
```

Comment:
So I'd like to add a line here like: 'The `stac` endpoint should function as a complete `Catalog` representation of all the data contained in the STAC API - all `Item`s should be linked in some way from that root through catalogs/collections, and as `item` links from those catalogs. But then I realized that sat-api isn't doing that yet. 

I feel like we should have an endpoint that stac browser can point at and navigate through down to the items in the same way you can with a static catalog. Though perhaps it makes sense to call that endpoint /browse or something, and then it'd be easier to not make it required?

Reply:
STAC Browser can now browse APIs because it can @mojodna added the ability to resolve links with rel=items as well as rel=item. 

I actually removed the existing paragraph about the API sub-catalogs and opened an issue for it:
https://github.com/radiantearth/stac-spec/issues/438

So in short, yes we should discuss how a browsable API should work and can add a better description of it, but we can make it another PR.